x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/trigger/schedule/tool/CronEvalToolTests.java

```diff
@@ -20,4 +24,28 @@ public void testParse() throws Exception {
         String output = execute(countOption, Integer.toString(count), "0 0 0 1-6 * ?");
         assertTrue(output, output.contains("Here are the next " + count + " times this cron expression will trigger"));
     }
+
+    public void testGetNextValidTimes() throws Exception {
+        {
+            String output = execute(
+                    "0 3 23 8 9 ? " + (Calendar.getInstance(TimeZone.getTimeZone("UTC"), Locale.ROOT).get(Calendar.YEAR) + 1));
+            assertTrue(output, output.contains("There are 1 times this cron expression will trigger:"));
```

Comment:
can you replace `assertTrue/assertFalse` with 

```
assertThat(output, containsString("FoObar"));
assertThat(output, not(containsString("ERROR")));
```

This way the assertion will return something helpful instead of just being true or false in order to ease debugging.